### DISCUSSION TITLE
I think the demo site is down

### DISCUSSION THREAD

**Discussion by MattPark** [LycheeOrg/Lychee#2502] [Q&A] [answered]
Appears demo site is down.  I tried to go to it to take a look.

<img width="1001" alt="image" src="https://github.com/user-attachments/assets/6bdd1b5f-d885-4970-82db-4bb4798a2760">

**Comment by ildyria** [comment]
ohoh. :(

**Reply by ildyria** [reply]
I got this from fly.io
![image](https://github.com/user-attachments/assets/f2cfc0c1-92d3-4d03-a543-ccb150940352)

**Reply by ildyria** [reply, marked as answer]
https://lychee-demo.fly.dev/landing back online.
